### PR TITLE
Refactor: Change util module to util package

### PR DESCRIPTION
**Pull request by breeswish**
1. Change `util` from module to package
2. Apply strict lint rules for `util`
3. Fix https://github.com/pingcap/tidb-dashboard/issues/1024

### CODE REVIEW

**Comment by breeswish** [comment]
@shhdgit @baurine PTAL, thanks. This PR nearly doesn't touch anything in the current hot path.

File: util/gormutil/virtualview/schema.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ func parseViewModelSchema(model interface{}) (viewSchema, error) {
 		ft := vt.Field(i)
 		props, err := decodeField(ft)
 		if err != nil {
-			return viewSchema{}, fmt.Errorf("field %s is invalid: %v", ft.Name, err)
+			return viewSchema{}, fmt.Errorf("field %s is invalid: %w", ft.Name, err)
```
Comment (shhdgit) [inline util/gormutil/virtualview/schema.go:146]: knowlege++

**Review by shhdgit — APPROVED** [review]
So many lint magic, lol.

**Pull request merged**